routes/users_src.js

```diff
@@ -0,0 +1,36 @@
+var fs = require("fs"); // To require File System module to use it's functions
+
+//This function will read the JSON file and will give the output
+exports.getUsers = function(req, res) {
+   fs.readFile( "./data/datapoint.json", 'utf8', function (err, data) {
+      if(data)
```

Comment:
```if``` condition should be always ```if (data) { ``` you need to have space and ``` { ``` on the same line. 
Javascript behaves differently when you have ```{``` on the next line.